src/gluonts/model/naive_2/_estimator.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# First-party imports
+from gluonts.core.component import validated
+from gluonts.model.estimator import DummyEstimator
+
+# Relative imports
+from ._predictor import Naive2Predictor
+
+
+class Naive2Estimator(DummyEstimator):
```

Comment:
I think this class isn't needed. We can use the predictor as forecaster directly.

Reply:
> Can we split the OWA metric and the model in two PRs?
> 
> And, I would prefer a different folder for this. Maybe `model/m4`?

But why? It's just another model like seasonal naive. This is also the reason why there is this `Naive2Estimator` class analogous to `SeasonalNaiveEstimator`.

Reply:
I think we should remove the `SeasonalNaiveEstimator` as well then.

Models which don't train should not have `Estimator`s.

Reply:
Overall I agree with not having an `Estimator` for models that are not trained. However, This can be left for another PR where these two, and possibly other models? are refactored accordingly?

Reply:
No.

Reply:
Having the `Estimator` class streamlines the testing of the models though.

Reply:
I think it's fine to have this class there since other models have the same, @jaheba we can open a new PR to remove these if we think they are redundant. case closed!

Reply:
Two wrongs don't make a right.

Is there any other reason keeping these, other than that existing models also do this?

Reply:
Ok, im working on removing them.